conf/script/build-cortx-ha

```diff
@@ -78,6 +88,7 @@ init() {
 
 cleanup() {
     echo "Delete resources"
+    log "${FUNCNAME[0]}: Delete resources"
     resources=(
```

Comment:
There is `for` loop below where each resource is being deleted. Please add log message to indicate which exactly resource is being deleted at this moment. This makes easier to understand on which resource script could fail.

Reply:
okay